discovery-infra/delete_nodes.py

```diff
@@ -43,7 +43,8 @@ def delete_nodes(tfvars):
             virsh_cleanup.DEFAULT_SKIP_LIST,
             [
                 tfvars.get("cluster_name", consts.TEST_INFRA),
-                tfvars.get("libvirt_network_name", consts.TEST_INFRA),
+                tfvars.get("libvirt_network_name", consts.TEST_INFRA_NETWORK),
+                tfvars.get("libvirt_network_name_2", consts.TEST_INFRA_NETWORK_2),
```

Comment:
not sure _2 is a good name, can't we have something more descriptive ? :)

Reply:
i am opened to suggestions

Reply:
I don't like it either.. Can it be an array?

Reply:
@YuviGold what do you mean by array? We will still need explicit names for each one

Reply:
it can be array in terraform but it will be much more complicated to implement i think

Reply:
> @YuviGold what do you mean by array? We will still need explicit names for each one

You said you want to support multiple networks

Reply:
after discussion with @ronniel1  i am cancelling this pull request, and  the second interface will be added by @RazRegev as part of his task